library/c/api_lightpush.go

```diff
@@ -6,7 +6,7 @@ package main
 import "C"
 import "github.com/waku-org/go-waku/library"
 
-// Publish a message using waku lightpush. Use NULL for topic to use the default pubsub topic..
+// Publish a message using waku lightpush. Use NULL for topic to derive the pubsub topic from the contentTopic.
```

Comment:
Hm. I have to take this into account for the rust bindings, since it's a breaking change (NULL = DefaultWakuTopic in current version)

Reply:
I am wondering if we can have some github action that would open an issue in rust-bindings as and when a change is made in c-bindings so that we won't miss such things.
For now, i will create an issue in rust-bindings for autosharding support that takes care of all such items

Reply:
Created an issue https://github.com/waku-org/waku-rust-bindings/issues/73